src/parser.ts

```diff
@@ -532,9 +532,11 @@ function parseMultipleSwitchFromToolArguments(args: string, sw: string, removeSu
     //    (example): -DMY_DEFINE='"SOME_VALUE"'
 
     function anythingBetweenQuotes(fullyQuoted: boolean): string {
-        let anythingBetweenReverseQuote: string = '\\`[^\\`]*?\\`';
-        let anythingBetweenSingleQuote: string = "\\'[^\\']*?\\'";
-        let anythingBetweenDoubleQuote: string = '\\"[^\\"]*?\\"';
+       // In regexp101:
+       // \\\"((?!\\\").)*\\\"|\"(\\\"|[^\"])*?[^\\\"]?\"
+        let anythingBetweenReverseQuote: string = '\\\\\\`((?!\\\\\\`).)*\\\\\\`|\\`(\\\\\\`|[^\\`])*?[^\\\\\\`]?\\`';
+        let anythingBetweenSingleQuote: string = "\\\\\\`((?!\\\\\\`).)*\\\\\\`|\\`(\\\\\\`|[^\\`])*?[^\\\\\\`]?\\`";
```

Comment:
This looks like a typo.  I expected to see single quotes, but I see backticks instead.

Reply:
The same pattern for single/double quotes and back ticks.
This PR is not done. I want to add a comment explaining the pattern and also maybe merge the 3 because it's the same pattern applied to ' and " and ` identically.

Reply:
Oh yes there was a typo. I noticed it now when I refactored the common code. Fixed now.